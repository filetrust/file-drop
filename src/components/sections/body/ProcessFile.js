import React from "react";
import { CSSTransition } from "react-transition-group";

import { RenderResults } from "../results";

export default function ProcessFile({ state } ) {
    const {fileProcessed } = state;
    return (
        <section className="process-file">
{/*
            <h1>Drag and drop a file to have it processed by the Glasswall d-FIRST&trade; Engine</h1>
            <h3>This free service is currently limited to a maximum file size of 6MB</h3>
*/}
            <CSSTransition in={fileProcessed} timeout={{enter: 500, exit: 500}} classNames="results">
                <RenderResults key={5} state={state} />
            </CSSTransition>
            {/* <div class="terms-link" onClick={this.props.toggleModal}>
              <button>Terms &amp; Conditions</button>
            </div> */}
        </section>
    );
}
