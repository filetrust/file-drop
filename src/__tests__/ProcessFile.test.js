import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ProcessFile from "../components/sections/process-file/ProcessFile";

Enzyme.configure({adapter: new Adapter()});

describe("ProcessFile", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<ProcessFile />);
    expect(wrapper.exists()).toBe(true);
  });

  it("matches snapshot", () => {
    const wrapper = shallow(<ProcessFile />);
    expect(wrapper).toMatchSnapshot();
  });
});
