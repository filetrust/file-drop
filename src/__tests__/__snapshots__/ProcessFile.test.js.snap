// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProcessFile matches snapshot 1`] = `ShallowWrapper {}`;
